### PR TITLE
build: add back es module + transpiled output

### PR DESCRIPTION
**Pull request by Hotell**
as we discussed here https://github.com/skatejs/project/pull/2#discussion_r117809028

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,7 +58,7 @@
   "scripts": {
     "docs:build": "gitbook install && gitbook build",
     "docs:watch": "gitbook install && gitbook serve",
-    "prepublish": "nwb build && babel src --out-dir es --presets=stage-0,es2017,es2016",
+    "prepublish": "nwb build && babel src --out-dir es-next --presets=stage-0,es2017,es2016",
```
Comment (treshugart) [inline package.json]: 🚲 shed: es-next or es-latest?

Comment (Hotell) [inline package.json]: hmm I think `es-latest` is more appropriate, updating

**Comment by treshugart** [comment]
Are we okay to close this since NWB already does it?

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /coverage
 /demo/dist
 /es
+/es-next
```
Comment (treshugart) [inline .gitignore]: `es-latest` (will probably fix the build, too)

Comment (Hotell) [inline .gitignore]: Whoops. Fixed

**Pull request merged**